rpc/types/block.go

```diff
@@ -16,24 +17,25 @@ import (
 	"github.com/ethereum/go-ethereum/common/hexutil"
 
 	grpctypes "github.com/cosmos/cosmos-sdk/types/grpc"
-
-	ethermint "github.com/evmos/ethermint/types"
 )
 
 // BlockNumber represents decoding hex string to block values
 type BlockNumber int64
 
 const (
-	EthPendingBlockNumber  = BlockNumber(-2)
-	EthLatestBlockNumber   = BlockNumber(-1)
-	EthEarliestBlockNumber = BlockNumber(0)
+	EthSafeBlockNumber      = BlockNumber(-4)
```

Comment:
This is not relevant to our chain as we have instant finality. So `Safe` `Finalized` and `Latest` should be BlockNumber(-1).